Orange/data/io.py

```diff
@@ -29,6 +29,9 @@
 from Orange.util import Registry, flatten, namegen
 
 
+__all__ = ["Flags"]
```

Comment:
FileFormat might be considered public as well, as someone might use it to define their own readers, but what else? Reader implementations are not supposed to be used directly, they are accessible through FileReader.get_reader.